tests/functional/tests.py

```diff
@@ -1910,6 +1936,10 @@ def main():
             log_output =  LogOutput(client.set_bucket_policy, 'test_set_bucket_policy_readonly')
             test_set_bucket_policy_readonly(client, log_output)
 
+            log_output = LogOutput(client.get_bucket_notification, 'test_get_bucket_notification')
```

Comment:
It looks like the test has been added only into `QuickMode`.
We also  need to add the same test into `FullMode`.

Reply:
@ebozduman I have added the test to full mode too.